src/app/[slug]/route.ts

```diff
@@ -4,11 +4,14 @@
 
 import { NextRequest, NextResponse } from "next/server";
 import { vers } from "../../controllers/dockerflow.js";
+import { getBreaches, getBreachIcons } from "../functions/server/getBreaches";
 
 export async function GET(req: NextRequest) {
   // heartbeat route for dockerflow
   const slug = req.nextUrl.pathname;
   if (slug.includes("__heartbeat__") || slug.includes("__lbheartbeat__")) {
+    // Ensure breaches and their icons are loaded after app startup
+    getBreaches().then(breaches => getBreachIcons(breaches));
```

Comment:
Although it shouldn't take our server down (cache).. it's absolutely the wrong place. Let's mark it a `TODO: remove ... `

Reply:
Q: How frequent is OPs/SRE fetching the `/__heartbeat__` and/or `/__lbheartbeat__` endpoints?

Reply:
I think we may want to ping SRE folks on this change. I quickly chatted w/ one unnamed SRE source who said:

> ... you’ll want to do very little actual work in both [routes]
> Doing quick checks like “is the database up” is reasonable

Reply:
@rhelmer 

I explicitly did not `await` this here to prevent delaying sending the `Response`, but obviously that is not clear, so good callout - I added a comment to make that explicit in 4b9a45841.

@mansaj 

Yes, this is very much the wrong place and I felt a bit dirty typing it. Added a TODO (a60fdd686d720cbfaf7252132954960f7da645f2 1164ef023) and MNTOR-1839.

@pdehaan 

Very frequently, but `getBreaches` and `getBreachIcons` include checks to ensure they only fetch data once.